server/boutique-hotel-technikum/src/main/java/com/example/boutiquehoteltechnikum/models/GuestEntity.java

```diff
@@ -0,0 +1,27 @@
+package com.example.boutiquehoteltechnikum.models;
+
+import jakarta.persistence.*;
+import lombok.*;
+
+@Entity
+@Table(name = "Guests")
+@Getter
+@Setter
+@Builder
+@NoArgsConstructor
+@AllArgsConstructor
+public class GuestEntity {
+
+    @Id
+    @GeneratedValue(strategy = GenerationType.IDENTITY)
+    private int guestId;
+
+    @Column(nullable = false)
+    private String firstName;
+
+    @Column(nullable = false)
+    private String lastName;
+
+    @Column(nullable = false, unique = true)
+    private String email;
```

Comment:
Please add a Size check to prevent too long inputs:
```
@Size(max = 50, message = "Die Email darf nicht länger als 50 Zeichen sein!")
```

Additonally, since there is no login, I wouldn't make it unique.